prefect_saturn/core.py

```diff
@@ -5,14 +5,16 @@
 import hashlib
 
 from typing import Any, Dict, List, Optional, Union
+from importlib.metadata import version
 from requests import Session
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
 
 import cloudpickle
 from prefect import Flow
 from prefect.client import Client
 
+from packaging.version import Version, parse
```

Comment:
I'm unsure if I need to add `packaging` and `importlib-metadata` to setup or any other files to ensure they are installed.

Reply:
I think so